pyproject.toml

```diff
@@ -98,80 +106,75 @@ markers = [
 [tool.ruff]
 line-length = 110
 src = [
-    'src',
-    'tests',
-    'docs',
+    "src",
+    "tests",
+    "docs",
 ]
 
 [tool.ruff.lint]
 extend-select = [
-    'F',      # Pyflakes (part of default flake8)
-    'W', 'E', # pycodestyle (part of default flake8)
-    'I',      # isort (import sorting)
-    # 'N',      # pep8-naming
-    'D',      # pydocstyle (docstring style guide)
-    'UP',     # pyupgrade (upgrade code to modern python)
-    'YTT',    # flake8-2020 (system version info)
-    'ANN',    # flake8-annotations (best practices for type annotations)
-    'S',      # flake8-bandit (security checks)
-    'BLE',    # flake8-blind-except (prevent blind except statements)
-    'B',      # flake8-bugbear (prevent common gotcha bugs)
-    'A',      # flake8-builtins (prevent shadowing of builtins)
-    'C4',     # flake8-comprehensions (best practices for comprehensions)
-    'T10',    # flake8-debugger (prevent debugger statements in code)
-    'EM',     # flake8-errormessages (best practices for error messages)
-    'FA',     # flake8-future-annotations (correct usage future annotations)
-    'ISC',    # flake8-implicit-str-concat (prevent implicit string concat)
-    'ICN',    # flake8-import-conventions (enforce import conventions)
-    'G',      # flake8-logging-format (best practices for logging)
-    'INP',    # flake8-no-pep420 (prevent use of PEP420, i.e. implicit name spaces)
-    'PIE',    # flake8-pie (misc suggested improvement linting)
-    # 'T20',    # flake8-print (prevent print statements in code)
-    'PT',     # flake8-pytest-style (best practices for pytest)
-    'Q',      # flake8-quotes (best practices for quotes)
-    'RSE',    # flake8-raise (best practices for raising exceptions)
-    'RET',    # flake8-return (best practices for return statements)
-    'SLF',    # flake8-self (prevent private member access)
-    'SLOT',   # flake8-slots (require __slots__ for immutable classes)
-    'SIM',    # flake8-simplify (suggest simplifications to code where possible)
-    'TID',    # flake8-tidy-imports (prevent banned api and best import practices)
-    'TCH',    # flake8-type-checking (move type checking imports into type checking blocks)
-    'INT',    # flake8-gettext (when to use printf style strings)
-    # 'ARG',    # flake8-unused-arguments (prevent unused arguments)
-    'PTH',    # flake8-use-pathlib (prefer pathlib over os.path)
-    # 'ERA',    # eradicate (remove commented out code)
-    'PGH',    # pygrep (simple grep checks)
-    'PL',     # pylint (general linting, flake8 alternative)
-    'TRY',    # tryceratops (linting for try/except blocks)
-    'FLY',    # flynt (f-string conversion where possible)
-    'NPY',    # NumPy-specific checks (recommendations from NumPy)
-    'PERF',   # Perflint (performance linting)
-    'LOG',
-    'RUF',    # ruff specific checks
+    "F",
```

Comment:
All of the comments that got deleted here are obviously not necessary for the functioning of the file, but aren't they kind of useful to keep around for documentation purposes?

Reply:
I apologize, I overlooked your comments on Friday when merging. These were stripped by the auto-formatter and I didn't get to restore them; fixed in 92be82d